examples/websub-hub-client-sample/websub_hub_client_sample.metatags

```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates the usage of the WebSub Hub Client subscribe and unsubscribe to updates for a particular topic at the hub.
```

Comment:
Edit: This BBE demonstrates the usage of the WebSub Hub Client subscribe and unsubscribe to update for a particular topic at the hub.